src/components/Header/Header.module.css

```diff
@@ -1,20 +1,126 @@
 .header {
-    background-color: #f8f9fa;
-    padding: 10px 20px;
-    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
-    position: fixed;
-    width: 100%;
-    top: 0;
-    left: 0;
-    z-index: 1000;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 48px 24px;
+  background-color: var(--midnight-green);
+  color: var(--primary-white);
+}
+
+.hamburgerIcon {
+  width: 20px;
+  height: 17px;
+  cursor: pointer;
+}
+
+.sideDrawer {
+  position: fixed;
+  right: -100%;
+  width: 255px;
+  height: 100%;
+  background-color: var(--police-blue);
+  transition: left 0.3s ease-in-out;
+  z-index: 200;
+  color: var(--primary-white);
+  background-image: url("../../assets/bg-pattern-about-1-mobile-nav-1.svg");
+  background-position-x: calc(100% + 100px);
+  background-position-y: bottom;
+  background-repeat: no-repeat;
+}
+
+.open {
+  right: 0;
+}
+
+.nav {
+  display: none;
+}
+
+.mobileNav {
+  padding-block: 56px;
+  padding-right: 25px;
+  padding-left: 48px;
+}
+
+.mobileNav ul,
+.nav ul {
+  list-style: none;
+}
+
+.mobileNav li,
+.nav li {
+  margin: 24px 0;
+}
+
+.mobileNav a,
+.nav a {
+  color: var(--primary-white);
+  text-decoration: none;
+}
+
+.mobileNav a:hover,
+.nav a:hover {
+  color: var(--coral-color);
+}
```

Comment:
The active link styling is only applied on the mobile menu only but the design shows for the desktop as well.

Reply:
Ahh, I had fixed this in one version of my code but I guess I lost it in refactors. Thanks for spotting it!

Reply:
fixed in https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/pull/35/commits/573461c02d9d341d335fa3e0f2f76b5d94408a29